src/main/java/com/alvarium/hash/exceptions/HashException.java

```diff
@@ -0,0 +1,10 @@
+package com.alvarium.hash.exceptions;
```

Comment:
It's my sense that you could just put this in `package com.alvarium.hash`. The additional `exceptions` package seems redundant in terms of its naming -- `com.alvarium.hash.exceptions.HashException`

Reply:
fixed in 0d3b63b1f35965acf32b53e9300a6adbc68ea290.